examples/bri-3/components/workgroup/workgroup/workgroup.interface.ts

```diff
@@ -0,0 +1,32 @@
+import { Workstep } from '../workstep/workstep';
+import { BpiSubject } from '../identity/bpiSubject';
+import { Security } from '../policy/security';
+import { Privacy } from '../policy/privacy';
+import { Workflow } from '../workflow/workflow';
+
+export interface IWorkgroup {
+    id: string; // TODO: Add uuid after #491
+    name: string;
+    administrator: BpiSubject[];
+    securityPolicy: Security;
+    privacyPolicy: Privacy;
+    participants: BpiSubject[];
+    worksteps: Workstep[];
+    workflows: Workflow[];
+
+    addParticipants(bpiSubjects: BpiSubject[]): BpiSubject[];
+    getParticipantsById(ids: string[]): BpiSubject[];
+    getAllParticipants(): BpiSubject[];
+    updateParticipant(id: string, update:any): BpiSubject;
```

Comment:
@Kasshern should that  not also be participants and `string[]`?

Reply:
Changed to array. Change to Update methods are likely subject to larger alterations in the near future once further framework is elucidated and nature of updates is well defined.  Of course the ability to request multiple updates across multiple entities is ideally preferred .